### PR TITLE
Update a stale comment.

### PR DESCRIPTION
**Pull request by vext01**

### CODE REVIEW

File: ykllvmwrap/src/ykllvmwrap.cc [hunk 1]
```diff
@@ -394,14 +394,17 @@ extern "C" void *__ykllvmwrap_irtrace_compile(char *FuncNames[], size_t BBs[],
       }
 
       // If execution reaches here, then the instruction I is to be copied into
-      // JITMod. We now scan the instruction's operands checking that each is
+      // JITMod.
+      //
+      // FIXME: We now scan the instruction's operands checking that each is
```
Comment (ltratt) [inline ykllvmwrap/src/ykllvmwrap.cc:399]: "each is corresponding variable" doesn't parse.

Comment (vext01) [reply ykllvmwrap/src/ykllvmwrap.cc:399]: There's an unchanged line in-between, so the line reads:

> FIXME: We now scan the instruction's operands checking that each is defined in JITMod.

Comment (ltratt) [reply ykllvmwrap/src/ykllvmwrap.cc:399]: Mea culpa!

**Pull request merged**